packages/pkgs/components/Modal/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import * as React from 'react';
+
+import Box from '../Box';
+
+export type Props = {
+    children?: React.Node,
+    open?: boolean,
+};
+
+const Modal: React$AbstractComponent<Props, HTMLElement> = React.forwardRef<Props, HTMLElement>(({
+  children = null,
+  open = true,
+}: Props, ref) => {
+  const containerRef = React.useRef();
+  const activeRef = ref || containerRef;
+
+  const styles = {
+    container: {
+      width: '50%',
+      height: 'auto',
+      padding: '40px',
+      border: 'solid',
+      borderRadius: '30px',
+      display: 'flex-wrap',
+    },
+  };
+
+  return open && (
+    <>
+      <Box
```

Comment:
Just this should be enough and you can remove all the other ref related stuff. But depends what you're trying to accomplish

```suggestion
      <Box
        ref={ref}
```